packages/sdk-schema-generator/package.json

```
@@ -0,0 +1,42 @@
+{
+	"name": "@directus/sdk-schema-generator",
+	"version": "1.0.0",
+	"description": "TypeScript Schema Generator",
```

Let's make this more related to Directus, e.g.:
```suggestion
	"description": "TypeScript Schema Generator for Directus",
```